feature-pack/pom.xml

```diff
@@ -89,6 +89,33 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.10</version>
+                <executions>
+                    <execution>
+                        <id>copy</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.jboss.wise</groupId>
+                                    <artifactId>wise-gwt-gui</artifactId>
+                                    <version>${version.wise-gwt-gui}</version>
+                                    <type>war</type>
+                                    <overWrite>true</overWrite>
+                                    <outputDirectory>${project.build.directory}/wildfly-${project.version}/modules/system/layers/base/org/wildfly/extension/wise/main/deployments</outputDirectory>
+                                    <destFileName>wise.war</destFileName>
```

Comment:
i think this can be done in feature pack config file (feature-pack-build.xml)
with <copy-artifacts> notion which would than copy the war file where ever you want it to.